keras_cv/datasets/pascal_voc/segmentation.py

```diff
@@ -259,6 +259,7 @@ def _build_metadata(data_dir, image_ids):
     return result
 
 
+@tf.function(jit_compile=True)
```

Comment:
Maybe add a comment here for some data: w/ or w/o the change, what happens, i.e., compile time change, process time change, etc (though it's highly hardware dependent)

Reply:
Yes in theory we have git blame to got to this PR and then to the benchmark comment but not many users has this habit... :smiley_cat:

Reply:
Done. Added comment.